### PR TITLE
Add blog on Vim

### PR DESCRIPTION
**Pull request by iamshubhankarkhare**
#### Checklist

- [x] You have read the [Contribution Guidlines](https://github.com/LoginRadius/engineering-portal/blob/master/CONTRIBUTING.md) before creating this PR.
- [x] You created a branch from `dev` branch
- [x] Blog follows [Blog Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/GUIDELINES.md).
- [x] Checked for all grammatical errors in the blog.
- [x] Checked for any duplicate blog entry.
- [x] You PR is raised on `dev` branch and not on`master`.

##### Description of change
Added a new blog , topic being VIM: Getting started.

##### Author Details for Blog Post Entry
id: Shubhankar Khare
bio: A delligent and promp learner , Shubhankar Khare is a passionate full stack web developer from India. He is crazy about learning new technologies and is an open source enthusiast 
github: iamshubhankarkhare
linkedin: shubhankar-khare

### CODE REVIEW

**Comment by iamshubhankarkhare** [comment]
@ashish8947 @pareek-narendra Please review

**Comment by pareek-narendra** [comment]
Thanks @iamshubhankarkhare 
Thanks for contributing. Your PR is under review.

File: content/blog/vim-getting-started/index.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+ So basically we have standard keybindings like d for delete, c for change, f for find etc. and we can pair them with numbers to repeat that action n number of times. Moreover doubling a command like so `yy` makes the command operate on the entire line.
+
+
+So to sum up,
+>Vim is for **programmers who want to raise their game**. In the hands of an expert, It shreds text at the speed of thought.
+
+ #### Cool, how do I get started?
+ Well, there is a learning curve, but it's worth it. Although this steep curve is exaggerated in the online world and the **quitting vim joke** is getting very old ( It's:q btw ).
+
+ Install Vim `sudo apt-get vim or pacman -S vim` for Linux user and type `vim <filename>` in your terminal. Press `i` to go to insert mode. Type some gibberish. Press `:wq` to save and quit . `:q!` to save without quitting.
```
Comment (vyasriday) [inline content/blog/vim-getting-started/index.md:52]: @iamshubhankarkhare don't you think `!q` is for quitting without saving? Just wanted to clear the doubt here.

**Review by vyasriday — APPROVED** [review]
@iamshubhankarkhare Nice post. Glad to see a vim user here. 👍🏼

**Review by puneetsingh24 — APPROVED** [review]

**Comment by iamshubhankarkhare** [comment]
> @iamshubhankarkhare Nice post. Glad to see a vim user here. 👍🏼

Thanks @vyasriday . 
Thank you guys for accepting my blog

**Pull request merged**